mailpoet/assets/js/src/listing/notices.jsx

```diff
@@ -72,12 +72,33 @@ export function MailerError(props) {
       links[match],
       'text/xml',
     ).firstChild;
+
+    const listOfAttributeNames = link.getAttributeNames();
+    const allowedAttributesList = [
+      'target',
+      'rel',
+      'class',
+      'data-email',
+      'data-type',
+    ];
+
+    // include these custom attributes in the final link
+    const otherAttributes = listOfAttributeNames.reduce((acc, name) => {
+      if (allowedAttributesList.includes(name)) {
+        // react requires the class attribute to be named className.
+        return {
+          ...acc,
+          [name === 'class' ? 'className' : name]: link.getAttribute(name),
+        };
+      }
+      return { ...acc };
```

Comment:
No need to create another object here, I guess you can simply return `acc`, what do you think?